docs/manage/fleet/locations.md

```diff
@@ -71,18 +68,20 @@ The location's page lists newly added organization, along with the organization
 
 You can remove any organization except the primary owner from the shared list by clicking the **X** to the right of the location in the shared list.
 
+<!-- location keys are going away but no new solution yet
```

Comment:
I'm commenting out the option to share location keys to share a location. Will there be location api keys? Or will that just no longer be possible?

Reply:
Yeah api keys can have authorizations scoped at the organization, location or robot level. right now, the only autogenerated api key is the robot api key. you can create api keys on a organization, location, or robot though via the cli after https://viam.atlassian.net/browse/RSDK-4737 (although idk if the docs changes needed got triggered for that since i think our docs are still out of date)